GliaWidgets/View/Chat/Message/Content/Text/ChatTextContentView.swift

```diff
@@ -101,11 +101,30 @@ extension ChatTextContentView: UITextViewDelegate {
         in characterRange: NSRange,
         interaction: UITextItemInteraction
     ) -> Bool {
-        if URL.scheme == "tel" || URL.scheme == "mailto" {
-            return true
+        if URL.scheme == "tel" {
```

Comment:
looks like ideal place to convert this to switch. what do you think, does it make sense?
```swift
        switch URL.scheme {
        case "tel":
            openPhoneDialer(url: URL)
        case "mailto":
            openEmailClient(url: URL)
        default:
           return false
        }
```